client/src/+assets/user-publish-steps/files-handler.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios'
 import axiosRetry from 'axios-retry'
 import { gatewayUri, filecoinUploadUri, ipfsGatewayUri } from 'src/config'
-import  {AssetFile} from '@nevermined-io/components-catalog/dist/node/types'
+import { AssetFile } from '@nevermined-io/catalog-core/dist/node/types'
```

Comment:
This is more for the catalog, not this pr

we should think of maybe exporting this type so that we don't have to go into `/dist/node/types`?